modules/installation-aws-permissions.adoc

```diff
@@ -152,6 +152,11 @@ If you use an existing Virtual Private Cloud (VPC), your account does not requir
 * `iam:TagInstanceProfile`
 * `iam:TagRole`
 
+[NOTE]
+=====
+If you specify an existing IAM role in the `install-config.yaml` file, the following IAM permissions are not required: `iam:CreateRole`,`iam:DeleteRole`, `iam:DeleteRolePolicy`, and `iam:PutRolePolicy`.
+=====
+
 [NOTE]
```

Comment:
We have two `[NOTE]` admonitions back-to-back here, which we really shouldn't do unless necessary. I'm not sure if that can be worked around here.